### ISSUE TITLE
Add `innermost` column for nested polygon

### ISSUE THREAD

**Issue by rustprooflabs** [rustprooflabs/pgosm-flex#50]
# Details

Follow up to #37.  Initial structure can find the outer boundaries with `nest_level = 1`.  Need `innermost` column since nesting levels can vary greatly.  

This seems to do the trick, need to test on larger scale data before implementing.  Will go into the table and index.

```sql
SELECT CASE WHEN osm_id_path[array_length(osm_id_path, 1)] = osm_id THEN True
            ELSE False
        END AS innermost,
        *
    FROM osm.place_polygon_nested 
;
```

**Comment by rustprooflabs** [comment]
The initial code example above worked at a per-row level and is the `row_innermost` generated column.  Final calculation of `innermost` column cannot happen until every row has its path calculated, at that point simple exclusion to determine true `innermost`.

**Comment by rustprooflabs** [comment]
Resolved in commit 67bb51c
